core/primitives/src/transaction.rs

```diff
@@ -61,8 +61,10 @@ pub struct SignedTransaction {
     pub transaction: Transaction,
     pub signature: Signature,
     #[borsh_skip]
+    #[serde(skip_deserializing)]
```

Comment:
Why only skip deserializing? Why not also skip serializing? (There should be a single param to skip both).